lib/liquidTestRunner.js

```diff
@@ -409,17 +409,27 @@ async function runTestsWithOutput(
 // RETURN (AND LOG) ONLY PASSED OR FAILED
 // CAN BE USED BY GITHUB ACTIONS
 async function runTestsStatusOnly(firmId, handle, testName = "") {
+  let status = "FAILED";
   const testResult = await runTests(firmId, handle, testName, false, "none");
```

Comment:
If the YAML file has only one empty row, then it will be `undefined` as you expect.
But, if the YAML has two or more empty rows, then it will no longer be undefined, returning a FAILED.

Example with two empty rows :thinking: 
```
{
  testRun: {
    status: 'test_error',
    error_message: 'Test was not found. Please make sure to put the cursor on your test definition line',
    error_line_number: 1
  },
  previewRun: null
}
```

Reply:
Nice find! I've added a trim() to the so if there are only spaces or newlines, it removes them and the testContext remains empty. This error might still occur though if all you have are comments in the file.